pages/linux/uuid.md

```diff
@@ -0,0 +1,32 @@
+# uuid
+
+> Generate and decode Universally Unique Identifiers (UUID).
+> More information: <https://manned.org/uuid>.
+
+- Generate a v1 UUID (time and node based):
+
+`uuid`
+
+- Generate a v4 UUID (random data based):
+
+`uuid -v {{4}}`
+
+- Generate a v3 UUID (name based) with an URL namespace:
+
+`uuid -v {{3}} ns:URL {{https://example.com}}`
```

Comment:
- Generate a v5 UUID (name based) with an URL namespace:

`uuid -v {{5}} ns:URL {{https://example.com}}`

Reply:
Not sure what I did here yet. I had problems synchronizing because of the switch from `master` to `main` branch that was finalized recently (good job btw).

I wantent to use GitHub UI to add changes. The reason:

UUID v3 uses MD5 and UUID v5 uses SHA1, which is preferred. Our examples here should only include UUID v1, v4 and v5 in the final form (v2 not available and v3 is less secure).

Reply:
It doesn't look like the suggestions you all leave here, that are ready to be committed. Can you guys give me a hand (replacing number 3 with the number 5) till I figure it out?

Reply:
@peterbabic you only have to put a

<p>```suggestion</p>

<p>```
around it

Reply:
@marchersimon that would have never occurred to me by chance